stories/core/Tree.stories.tsx

```diff
@@ -0,0 +1,139 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import { Tree, TreeProps, TreeNode, Checkbox } from '../../src/core';
+import { action } from '@storybook/addon-actions';
+import { SvgPlaceholder } from '@itwin/itwinui-icons-react';
+import { NodeData } from '../../src/core/Tree/Tree';
+
+export default {
+  component: Tree,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+  },
+  title: 'Core/Tree',
+} as Meta<TreeProps<unknown>>;
+
+export const Basic: Story<TreeProps<unknown>> = () => {
+  type StoryData = {
+    id: string;
+    label: string;
+    subLabel: string;
+    subItems: StoryData[];
+  };
+
+  const [selectedNodes, setSelectedNodes] = React.useState([
+    'Node 0',
+    'Node 3.2',
+    'Node 22',
+  ]);
+  const onSelectedNodeChange = React.useCallback(
+    (nodeId: string, selected: boolean) => {
```

Comment:
I guess its css bug, but I really dont like that when you click it gets focus style and released it dissapears... It is really weird and not the first component I noticed.
Also onclick it stays focused so if you have 2 selected nodes near, it gets this double border or smth..

Reply:
Focus bug should be fixed with this pr https://github.com/iTwin/iTwinUI/pull/456

Reply:
I guess it is fixed, but we still have double border..
![image](https://user-images.githubusercontent.com/81580355/151307062-ebddf1f0-57cb-4f82-b3ea-0e46ae723ad0.png)

We can fix that later, I assume